src/core/Button/styles.css

```diff
@@ -22,11 +24,16 @@
    icon get's pointer-events: none */
 .button i {
     pointer-events: none;
-    margin-right: 5px;
-    margin-bottom: 2px;
+    margin-right: 8px;
+    margin-left: -4px;
```

Comment:
can we do this without -ive margins

Reply:
![no-ive-margins](https://user-images.githubusercontent.com/353236/47025492-e3e1e580-d163-11e8-9324-9f437938ce53.png)
Removing the margins looks pretty bad. I experimented a little with some flexbox attributes (i.e. `justify-content: space-between;` etc.) but couldn't achieve the same result.

Again, suggestions are welcome.